src/docs/product/dashboards/widget-builder/index.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Widget Builder
+sidebar_order: 20
```

Comment:
This just needs to be between the number on the Custom Dashboards page (10) and the one on the Widget Library page (20). 

```suggestion
sidebar_order: 15
```